docs/preview/01-index.md

```diff
@@ -5,20 +5,18 @@ slug: /
 sidebar_label: Welcome
 ---
 
+# Introduction
+
+Arcus EventGrid allows you to work more easily with Azure EventGrid during event publishing and parsing. Instead of worrying about the event type (EventGrid events, CloudEvent events, custom events), Arcus EventGrid allows you to interact with events in a simple manner by parsing the events to a canonical format. Same goes with publishing events where Arcus EventGrid lets you publish in a secure and transient fashion events on EventGrid.
```

Comment:
How does it help in parsing ?  You mean at the receiving end ?

Reply:
Yes, we have a class called `EventParser` that parsers any kind of event from EventGrid to a canonical format. So, the library can both be used for publishing (sending) and parsing (receiving).